core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleParser.java

```diff
@@ -1318,4 +1319,31 @@ protected void throwEOFException() throws RDFParseException {
 	protected int getLineNumber() {
 		return lineNumber;
 	}
+
+	private StringBuilder getBuilder() {
+		parsingBuilder.setLength(0);
+		return parsingBuilder;
+	}
+
+	/**
+	 * Appends the characters from codepoint into the string builder. This is the same
+	 * as Character#toChars but prevents the additional char array garbage for BMP codepoints.
+	 * @param dst the destination in which to append the characters
+	 * @param codePoint the codepoint to be appended
+	 */
+	private static void appendCodepoint(StringBuilder dst, int codePoint) {
+		/*
+		 * This code is copy pasted from Character#toChars but doesn't allocate new arrays.
```

Comment:
Someone at Eclipse pointed out that you really only copy 3 lines of code and that is so trivial that it is unlikely to cause a problem. So I'm fine with just including this. However, to avoid future confusion/questions, can you remove this comment about it being "copy pasted"?